doc/source/workflows/basics.rst

```diff
@@ -256,8 +284,10 @@ Note that tasks that have side effects still need to be idempotent. This is beca
     # UNSAFE: we could book multiple flight tickets
     workflow.run(book_flight_unsafe.bind())
 
-.. code-block:: python
-    :caption: Idempotent workflow:
+Idempotent workflow:
+
+.. testcode::
+    :skipif: True
```

Comment:
Why do we skip this? Is it expected to error?

Reply:
It's an incomplete code that won't compile.

```
 @ray.remote
    def book_flight_idempotent(request_id: str) -> FlightTicket:
       if service.has_ticket(request_id):
           # Retrieve the previously created ticket.
           return service.get_ticket(request_id)
       return service.book_flight(request_id)
```

We don't have a `service` here.

Reply:
Got it!

Reply:
Got it!